### PR TITLE
Updated Pagerduty Self Service GitHub Actions

### PR DESCRIPTION
**Pull request by mk-armah**
# Description
Improved existing pagerduty guides and fixed bug in obtaining the entity ids

## Updated docs pages
/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/

### BugFix  
 - correct entity identifer in port action to `.entity.identifier`

### Improvement
- Added `relations` field to all DAY-2 actions
- Improved `create-pagerduty-service` and `trigger incident`  actions to report updated runs back to port

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/acknowledge-incident.md [hunk 1]
```diff
@@ -191,8 +192,9 @@ Create a new self service action using the following JSON configuration.
       "{{if (.inputs | has(\"from\")) then \"from\" else null end}}": "{{.inputs.\"from\"}}",
```
Comment (pazhersh) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/acknowledge-incident.md:192]:
```suggestion
      "from": "{{.inputs.\"from\"}}",

```

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/change-on-call-user.md [hunk 2]
```diff
@@ -250,16 +251,17 @@ Create a new self service action using the following JSON configuration.
     "type": "GITHUB",
     "org": "<GITHUB_ORG>",
     "repo": "<GITHUB_REPO>",
-    "workflow": "change-incident-owner.yaml",
+    "workflow": "change-on-call-user.yaml",
     "workflowInputs": {
       "{{if (.inputs | has(\"ref\")) then \"ref\" else null end}}": "{{.inputs.\"ref\"}}",
       "{{if (.inputs | has(\"start_time\")) then \"start_time\" else null end}}": "{{.inputs.\"start_time\"}}",
       "{{if (.inputs | has(\"end_time\")) then \"end_time\" else null end}}": "{{.inputs.\"end_time\"}}",
       "{{if (.inputs | has(\"new_on_call_user\")) then \"new_on_call_user\" else null end}}": "{{.inputs.\"new_on_call_user\"}}",
```
Comment (pazhersh) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/change-on-call-user.md:259]: all the inputs here too could go through this change:
```suggestion
      "new_on_call_user": "{{.inputs.\"new_on_call_user\"}}",

```

Comment (pazhersh) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/change-on-call-user.md:259]: Thats relevant for the other files in this PR as well

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-service.md [hunk 3]
```diff
@@ -83,17 +98,37 @@ jobs:
...
         id: fetch_oncalls
         uses: fjogeleit/http-request-action@v1
         with:
-          url: 'https://api.pagerduty.com/oncalls?include[]=users&escalation_policy_ids[]=${{ fromJson(github.event.inputs.escalation_policy).id }}'
+          url: 'https://api.pagerduty.com/oncalls?include[]=users&escalation_policy_ids[]=${{ inputs.escalation_policy }}'
           method: 'GET'
           customHeaders: '{"Content-Type": "application/json", "Accept": "application/json", "Authorization": "Token token=${{ secrets.PAGERDUTY_API_KEY }}"}'
 
-      - name: Create a log message
+      - name: Extract User Emails
```
Comment (pazhersh) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-service.md:111]: Did you copy these steps from somewhere or are they completely new?

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/trigger-pagerduty-incident.md [hunk 4]
```diff
@@ -76,14 +76,68 @@ jobs:
...
-                "severity": ${{ github.event.inputs.severity }}, 
+                "summary": "${{ github.event.inputs.summary }}",
+                "source": "${{ github.event.inputs.source }}",
+                "severity": "${{ github.event.inputs.severity }}"
               },
               "event_action": "${{ github.event.inputs.event_action }}",
               "routing_key": "${{ github.event.inputs.routing_key }}"
             }
-      - name: Create a log message
+      - name: Get PagerDuty Incident Details
```
Comment (pazhersh) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/trigger-pagerduty-incident.md:86]: here too, are the new steps in this pr copied from another example we have, or are they completely new here?

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-service.md (hunk 3 above)
Comment (mk-armah) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-service.md:111]: i'm not sure i understand

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/trigger-pagerduty-incident.md (hunk 4 above)
Comment (mk-armah) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/trigger-pagerduty-incident.md:86]: Its new. I just extended the capabilities of the action to report back to port. it was not so initially

**Review by yairsimantov20 — CHANGES_REQUESTED** [review]
go over @pazhersh comments

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-service.md (hunk 3 above)
Comment (pazhersh) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-service.md:111]: allright, cool

**Review by pazhersh — APPROVED** [review]

**Review by yairsimantov20 — APPROVED** [review]

**Pull request merged**